src/core/openthread.cpp

```diff
@@ -1636,6 +1636,21 @@ ThreadError otSetActiveDataset(otInstance *aInstance, const otOperationalDataset
     return error;
 }
 
+bool otIsNodeCommissioned(otInstance *aInstance)
+{
+    otOperationalDataset aDataset;
+
+    otGetActiveDataset(aInstance, &aDataset);
```

Comment:
The downside of removing the `namespace` wrapper in `openthread.cpp` is that the full namespace hierarchy needs to be specified for all C++ names.